src/cpp/Directory.cpp

```diff
@@ -8,26 +8,29 @@
 #include "Directory.h"
 
 
-Directory::Directory(const std::string &title): KnowledgeItem{title} {

```

Comment:
Please avoid mixing meaningful changes and reformatting/refactoring/... in a single commit
This complicates review and increases probability of missing something important
Better to create a new PR for minor changes - it could be accepted quickly and then everyone will be using proper version
(This PR probably shouldn't be changed - but in future please follow best practices)